core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.ts

```diff
@@ -1,37 +1,58 @@
 import { EditorModule, TINYMCE_SCRIPT_SRC } from '@tinymce/tinymce-angular';
-import { RawEditorOptions } from 'tinymce';
+import { RawEditorOptions, TinyMCE } from 'tinymce';
 
 import { ChangeDetectionStrategy, Component, Input, inject, signal } from '@angular/core';
 import { ControlContainer, FormsModule, ReactiveFormsModule } from '@angular/forms';
 
+import { DialogService } from 'primeng/dynamicdialog';
+
 import { DotCMSContentTypeField } from '@dotcms/dotcms-models';
 
+import { DotWysiwygPluginService } from './dot-wysiwyg-plugin/dot-wysiwyg-plugin.service';
+
+declare global {
+    interface Window {
+        tinymce: TinyMCE;
+    }
+}
```

Comment:
If we have another `declare global` in ant other place of the app, doesn't those get's overwritten by eachother?

Reply:
No needed. I forgot to remove this when trying a different approach.